### PR TITLE
Home Task 10 - JS_promises

### PR DESCRIPTION
**Pull request by Sergii112**

### CODE REVIEW

File: homeTask_10/task1.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+function busSchedule(message, ms) {
```
Comment (sttaran) [inline homeTask_10/task1.js]: function name should represent an action - "What to do?"

File: homeTask_10/task2.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+function fetchToDo() {
+  return new Promise((resolve, reject) => {
```
Comment (sttaran) [inline homeTask_10/task2.js]: remove this Promise, fetch already return Promise

File: homeTask_10/task2.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    fetch(`https://jsonplaceholder.typicode.com/todos/1`)
+      .then((response) => response.json())
+      .then(toDo => resolve(toDo))
+      .catch(error => reject(error))
+  })
+}
+
+function fetchUser() {
+  return new Promise((resolve, reject) => {
```
Comment (sttaran) [inline homeTask_10/task2.js]: same here

**Review by sttaran — CHANGES_REQUESTED** [review]

File: homeTask_10/task2.js (hunk 2 above)
Comment (Sergii112) [reply homeTask_10/task2.js]: done

File: homeTask_10/task2.js (hunk 3 above)
Comment (Sergii112) [reply homeTask_10/task2.js]: done

File: homeTask_10/task1.js (hunk 1 above)
Comment (Sergii112) [reply homeTask_10/task1.js]: done

**Pull request merged**